selenium/che-selenium-test/src/test/resources/suites/PassingSuite.xml

```diff
@@ -0,0 +1,775 @@
+<!--
```

Comment:
What is this suite for? We have already have CheSuite.xml, and it is using by default.

Reply:
removed